twitter-openapi-typescript/src/apis/userApi.ts

```diff
@@ -33,7 +33,7 @@ export class UserApiUtils {
 
   async getUserByScreenName(param: getUserByScreenNameParam): Promise<UserApiUtilsResponse> {
     const args = {
-      screenName: param.screenName,
+      screen_name: param.screenName,
```

Comment:
`screenName` is not recognized by Twitter. I looked in the request variables and apparently Twitter uses `screen_name` instead.